nservicebus/sagas/index.md

```diff
@@ -21,6 +21,7 @@ Design processes with more than one remote call to use sagas.
 
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
+NOTE: If a message is received while there is not saga instance then by default this message is discarded and succesfully processed. If the received message must not be discarded and throw an error so that it gets moved to the error queue then it is required to override saga not found behavior by implementing `IHandleSagaNotFound` and throw an exception. 
```

Comment:
We need to mention that this only happens if the message isn't configured to start a new saga.

Perhaps:

>Messages not configured to start new saga instances will be discarded and considered successful if no existing instance is found. If the received message must not be discarded..

Thoughts?

Reply:
> If a message is received while there is ~~not~~ no saga instance then by default this message is discarded and succesfully processed.

> If the received message must not be discarded and throw an error so that it gets moved to the error queue then it is required to override saga not found behavior by implementing `IHandleSagaNotFound` and throw an exception. 

that sentence is too complex. I think this would require a larger section with a corresponding snippet.